docs/setup/config.md

```diff
@@ -107,12 +109,20 @@ Please make sure to set the maximum ticket life span high long running jobs. The
 
 If you are on YARN, then it is sufficient to authenticate the client with Kerberos. On a Flink standalone cluster you need to ensure that, initially, all nodes are authenticated with Kerberos using the `kinit` tool.
 
+### Resource Manager
+
+- `resourcemanager.rpc.port`: The config parameter defining the network port to connect to for communication with the resource manager.
```

Comment:
No, it doesn't. It is currently not in use (except for testing code). In Yarn, the port of the application master is always the port of the job manager because the two run in the same actor system.